js/weather/apikey.js

```diff
@@ -0,0 +1,3 @@
+export const config ={
+    apiKey : 'fe886d5e8237a6bd251e905317a17cfa'
+}
```

Comment:
It would be best to hide this API Key. Here is an article that talks about doing this on GitHub. https://sung.codes/blog/2017/07/08/hiding-api-keys-github/